docbrown/db/src/data.rs

```diff
@@ -26,3 +34,107 @@ fn fetch_file(name: &str, url: &str, hash: &str) -> Result<PathBuf, FetchDataErr
     fetch(url, hash, &file)?;
     Ok(file)
 }
+
+pub fn lotr_graph(shards: usize) -> GraphDB {
```

Comment:
please don't, use the CSVLoader that relies on Serde, there are examples

Reply:
https://github.com/Raphtory/docbrown/blob/main/examples/rust/src/bin/lotr/main.rs#L8